includes/class-wcpdf-settings.php

```diff
@@ -210,6 +210,9 @@ public function ajax_preview() {
 						}, 99, 2 );
 					}
 
+					// disable historical settings
+					add_filter( 'wpo_wcpdf_document_use_historical_settings', __return_false(), 999, 2 );
```

Comment:
White this works, it has the downside that it will store the settings prematurely for the order that was previewed:
https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/a6df9d5558626235e7632d347564df417c778767/includes/documents/abstract-wcpdf-order-document.php#L154-L159

This could be resolved by disabling storing settings:
https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/a6df9d5558626235e7632d347564df417c778767/includes/documents/abstract-wcpdf-order-document.php#L178

However, I wonder if we should just do something like this:

```php
$document->settings = ! empty( $document->latest_settings ) ? $document->latest_settings : $document->get_settings( true );
```
before generating the preview here:
https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/a6df9d5558626235e7632d347564df417c778767/includes/documents/abstract-wcpdf-order-document.php#L767-L777